spec/awesome_spawn_spec.rb

```diff
@@ -141,6 +141,20 @@
       it "contains #error" do
         expect(subject.send(run_method, "echo 'bad' >&2 && false").error).to eq("bad\n")
       end
+
+      it "combines output when using :capture2e" do
+        result = subject.send(run_method, "echo 'bad' >&2 && false", :capture2e => true)
+
+        expect(result.output).to eq("bad\n")
+        expect(result.error).to  eq("bad\n")
```

Comment:
Since error is being redirected into output, I don't think error should have anything in it, and I definitely don't think it should have STDOUT in it...Not sure if that means it should be `""` or `nil` or `"\n"`

Reply:
I think it is `""` when no `.error` exists normally, so probably just `""`.